Podcast/Player.swift

```diff
@@ -18,6 +18,19 @@ class Player: NSObject {
         autoplayEnabled = true
         currentItemPrepared = false
         isScrubbing = false
+        //play podcasts in background of app
+        do {
+            try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback, with: .mixWithOthers)
+            print("AVAudioSession Category Playback OK")
+            do {
+                try AVAudioSession.sharedInstance().setActive(true)
```

Comment:
We definitely don't want this active the whole time, only when actually playing. This should be toggled on play/pause.

Reply:
Whoops! did not mean to add this code to this PR.. I'll take it out for now since this is a whole separate issue